tools/run_tests/run_tests.py

```diff
@@ -759,9 +759,13 @@ def _start_port_server(port_server_port):
     env = dict(os.environ)
     env['BUILD_ID'] = 'pleaseDontKillMeJenkins'
     if platform.system() == 'Windows':
+      # Working directory of port server needs to be outside of Jenkins
+      # workspace to prevent file lock issues.
```

Comment:
Looks like it's failed: probably check the referenced log file for what went wrong

https://grpc-testing.appspot.com/job/gRPC_pull_requests/4117/config=dbg,language=c,platform=windows/console